.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Change Summary
+
+<!-- Please give a short summary of the changes. -->
+
+## Related issue number
+
+<!-- Are there any issues opened that will be resolved by merging this change? -->
+<!-- WARNING: please use "fix #123" style references so the issue is closed when this PR is merged. -->
+
+## Checklist
+
+* [ ] The pull request title is a good summary of the changes
+* [ ] Documentation reflects the changes where applicable
+* [ ] Unit tests for the changes exist
+* [ ] Tests pass on CI and locally
```

Comment:
Checklist looks good but could be even more specific:
Before making a pull request, ensure that tests pass locally. (Don't overload github-CI)
Make the PR a draft-PR (Under reviewers).
After CI passes satisfactorily, make PR ready to review (select reviewer)

Reply:
> Before making a pull request, ensure that tests pass locally. (Don't overload github-CI)

Sometimes we open draft PRs well before any intention of merging. [Draft PRs are really designed to be opened as soon as one has an idea.](https://github.blog/2019-02-14-introducing-draft-pull-requests/) So while I agree testing locally should be done in conjunction with opening a PR, I don't see a reason to not open a draft PR if you expect tests to fail on CI.

> Make the PR a draft-PR (Under reviewers).
After CI passes satisfactorily, make PR ready to review (select reviewer)

:+1:  I will write something to this effect.